chain/jsonrpc/openapi/src/main.rs

```diff
@@ -576,192 +596,211 @@ struct RpcClientConfigRequest;
         &mut all_paths,
         "send_tx".to_string(),
         "Sends transaction. Returns the guaranteed execution status and the results the blockchain can provide at the moment.".to_string(),
     );
-    add_spec_for_path::<RpcStatusRequest, RpcStatusResponse>(
+    add_spec_for_path::<RpcStatusRequest, RpcStatusResponse, RpcStatusError>(
         &mut all_schemas,
         &mut all_paths,
         "status".to_string(),
         "Requests the status of the connected RPC node. This includes information about sync status, nearcore node version, protocol version, the current set of validators, etc.".to_string(),
     );
-    add_spec_for_path::<RpcTransactionStatusRequest, RpcTransactionResponse>(
+    add_spec_for_path::<RpcTransactionStatusRequest, RpcTransactionResponse, RpcTransactionError>(
         &mut all_schemas,
         &mut all_paths,
         "tx".to_string(),
         "Queries status of a transaction by hash and returns the final transaction result."
             .to_string(),
     );
-    add_spec_for_path::<RpcValidatorRequest, RpcValidatorResponse>(
+    add_spec_for_path::<RpcValidatorRequest, RpcValidatorResponse, RpcValidatorError>(
         &mut all_schemas,
         &mut all_paths,
         "validators".to_string(),
         "Queries active validators on the network. Returns details and the state of validation on the blockchain.".to_string(),
     );
-    add_spec_for_path::<RpcClientConfigRequest, RpcClientConfigResponse>(
+    add_spec_for_path::<RpcClientConfigRequest, RpcClientConfigResponse, RpcClientConfigError>(
         &mut all_schemas,
         &mut all_paths,
         "client_config".to_string(),
         "Queries client node configuration".to_string(),
     );
-    add_spec_for_path::<RpcStateChangesInBlockByTypeRequest, RpcStateChangesInBlockResponse>(
+    add_spec_for_path::<RpcStateChangesInBlockByTypeRequest, RpcStateChangesInBlockResponse, RpcStateChangesError>(
         &mut all_schemas,
         &mut all_paths,
         "EXPERIMENTAL_changes".to_string(),
         "[Deprecated] Returns changes for a given account, contract or contract code for given block height or hash. Consider using changes instead.".to_string(),
     );
-    add_spec_for_path::<RpcStateChangesInBlockRequest, RpcStateChangesInBlockByTypeResponse>(
+    add_spec_for_path::<RpcStateChangesInBlockRequest, RpcStateChangesInBlockByTypeResponse, RpcStateChangesError>(
         &mut all_schemas,
         &mut all_paths,
         "EXPERIMENTAL_changes_in_block".to_string(),
         "[Deprecated] Returns changes in block for given block height or hash over all transactions for all the types. Includes changes like account_touched, access_key_touched, data_touched, contract_code_touched. Consider using block_effects instead".to_string(),
     );
-    add_spec_for_path::<RpcCongestionLevelRequest, RpcCongestionLevelResponse>(
+    add_spec_for_path::<RpcCongestionLevelRequest, RpcCongestionLevelResponse, RpcCongestionLevelError>(
```

Comment:
this one is defined as
`pub type RpcCongestionLevelError = RpcChunkError;`
so it shows up as `RpcChunkError` in the schema, not sure if it is the expected behavior?